client/app/telecom/telephony/line/calls/callWaiting/telecom-telephony-line-calls-callWaiting.html

```diff
@@ -49,9 +49,7 @@ <h2 data-translate="loading"></h2>
                                 <i class="oui-icon oui-icon-success" aria-hidden="true"></i>
                                 <i class="oui-icon oui-icon-error" aria-hidden="true"></i>
                             </label>
-                        </div>
-                        <!-- /.widget-presentation -->
-
+                        </div><!-- /.widget-presentation -->
```

Comment:
Any reason? :smile:

Reply:
be uniform to most those type of comment